README.md

```diff
@@ -1 +1,221 @@
 # AIOShumway
+
+[![Build Status](https://travis-ci.org/spotify/aioshumway.svg?branch=master)](https://travis-ci.org/spotify/aioshumway) 
+
+A wrapper around the [shumway](https://github.com/spotify/shumway) library that uses [asyncio](https://docs.python.org/3/library/asyncio.html) to make all IO operations non-blocking and compatible with the async/await syntax.
+
+## Requirements
+
+* Python 3.6
+* Support for Linux & OS X
+
+## To Use
+
+```sh
+(env) $ pip install aioshumway
+```
+
+**NOTE:** This is a lightly touched up copy of [shumway's](https://github.com/spotify/shumway) README since the only difference between these libraries is that this one provides an awaitable interface.
```

Comment:
Is it a copy if you depend on it?

Reply:
I'm not sure I understand your suggestion, I'll ask you about it face to face tomorrow morning.

Reply:
I've moved the note up so that it's placement makes it more explicit what "copy" refers to: https://github.com/spotify/aioshumway/blob/async_shumway/README.md.